src/sparse/node.rs

```diff
@@ -160,11 +143,11 @@ impl Node {
     }
 
     pub fn is_leaf(&self) -> bool {
-        self.prefix() == LEAF || self.is_placeholder()
+        self.prefix() == Prefix::Leaf || self.is_placeholder()
     }
 
     pub fn is_node(&self) -> bool {
-        self.prefix() == NODE
+        self.prefix() == Prefix::Internal
```

Comment:
I think the old name `Node` was more descriptive than `Internal`=)

Reply:
I kinda like `Node` too, but I worry about creating confusion. Sometimes I use `Node` to refer to any node in the tree (including a `Leaf`); sometimes I use `Node` to mean an intermediate node in the tree that isn't a `Leaf`. I suppose it's not too bad, and we can usually understand the usage based on context. I will see if anyone else has comments on this, but if not, I will change it back to `Node`.